src/components/RequireAuth.js

```diff
@@ -204,12 +209,22 @@ const RequireAuth = (props) => {
   const modulesManager = useModulesManager();
   const auth = useAuthentication();
   const cfg = children.props.modulesManager.cfg;
+  const calendarSwitch = modulesManager.getConf(
+    "fe-core",
+    "allowSecondCalendar",
+    true,
+  );
+
   const isAppBarMenu = useMemo(() => theme.menu.variant.toUpperCase() === "APPBAR", [theme.menu.variant]);
+  const defaultValue = false;
+  const [isSecondaryCalendar, setSecondaryCalendar] = useBoolean(defaultValue);
+  const dispatch = useDispatch();
+
+  useEffect(() => {dispatch(toggleCurrentCalendarType(!isSecondaryCalendar))}, [isSecondaryCalendar])
```

Comment:
You can remove curly brackets here:
`useEffect(() => dispatch(toggleCurrentCalendarType(!isSecondaryCalendar)), [isSecondaryCalendar]);`